resources/webfonts/src/main.js

```diff
@@ -0,0 +1,171 @@
+var fontkit = require('fontkit');
```

Comment:
Add a readme.md in this folder explaining what this stuff is.

Reply:
Added very short description.